src/modules/facility-investor/facility-investor.controller.test.ts

```diff
@@ -0,0 +1,62 @@
+import { RandomValueGenerator } from '@ukef-test/support/generator/random-value-generator';
+
+import { CreateFacilityInvestorRequestItem } from './dto/create-facility-investor-request.dto';
+import { CreateFacilityInvestorResponse } from './dto/create-facility-investor-response.dto';
+import { FacilityInvestorController } from './facility-investor.controller';
+import { FacilityInvestorService } from './facility-investor.service';
+
+describe('FacilityInvestorController', () => {
+  const valueGenerator = new RandomValueGenerator();
+
+  let controller: FacilityInvestorController;
+
+  let facilityInvestorServiceCreateInvestorForFacility: jest.Mock;
+
+  beforeEach(() => {
+    const facilityInvestorService = new FacilityInvestorService(null, null);
+
+    facilityInvestorServiceCreateInvestorForFacility = jest.fn();
+    facilityInvestorService.createInvestorForFacility = facilityInvestorServiceCreateInvestorForFacility;
+
+    controller = new FacilityInvestorController(facilityInvestorService);
+  });
+
+  describe('createInvestorForFacility', () => {
+    const facilityIdentifier = valueGenerator.stringOfNumericCharacters();
```

Comment:
Can we add some negative assertions, where either one or multiple payload attributes do not contain valid values please.

Reply:
There is no logic for invalid values in the controller because the controller assumes the request body has already been validated by the framework and decorators

Reply:
That is, there isn't really anything about invalid values that we can test in these unit tests. But the API tests do check invalid request bodies (we can check it at the API level because the API tests _do_ include the validation logic from the framework and decorators)